### PR TITLE
March Into Mapness: Transit Booth

### PR DESCRIPTION
**Pull request by Hermod132**
## About The Pull Request
Adds a new space ruin, the Transit Booth. Designed to be a small respite from space travel, before sending intrepid explorers rapidly space-bound. Includes duty-free alcohol and books!

image
### Mapping March
I don't need anything from this, mapping itself is fun :)
Ckey to receive rewards: N/A

## Why It's Good For The Game
The March Into Madness event requested more space ruins, this would be a great way to increase the rate of exploration for these new ruins -- as well as being a decent place to be itself. Also I checked, the mass drivers do not table you, do not worry about high speed tablings.

Something to note is that there is another PR with a very similar premise to this, it is #74121. 
## Changelog
:cl:
add: Added more things
The space ruin "transit_booth.dmm"
/:cl:

Added "transit_booth.dmm" filepath to config/spaceruinblacklist.txt, due to the post https://github.com/tgstation/tgstation/pull/74110#pullrequestreview-1347641224
This is my first time working with github, pls be gentle     ; _ ;

### CODE REVIEW

File: _maps/RandomRuins/SpaceRuins/transit_booth.dmm [hunk 1]
```diff
@@ -0,0 +1,657 @@
...
+/area/ruin/space/has_grav)
+"h" = (
+/obj/effect/turf_decal/stripes/red/line,
+/obj/machinery/door/window/right/directional/west{
+	name = "Eastbound"
+	},
+/turf/open/floor/iron/dark,
+/area/ruin/space/has_grav{
+	name = "transitbooth"
+	})
```
Comment (Zonespace27) [inline _maps/RandomRuins/SpaceRuins/transit_booth.dmm:65]: this is your issue. NEVER varedit areas

Comment (Hermod132) [reply _maps/RandomRuins/SpaceRuins/transit_booth.dmm:65]: Okay, that makes sense. What should I do to create a unique area name?

Comment (Zonespace27) [reply _maps/RandomRuins/SpaceRuins/transit_booth.dmm:65]: Create an area subtype and set its name variable

**Review by Maurukas — CHANGES_REQUESTED** [review]
Two things before this is ready:

* Please replace the varedited area with a subtype in the appropriate file
* Please create a ruin datum in `code/datum/ruins/space.dm` so that your ruin will spawn in game. 

If you need help with either of these, ask in the mapping-general or mapping-event-help channels on discord.

**Comment by Hermod132** [comment]
I am uncertain how to fix the current Run Linters error "File is not included," I believe it has something to do with checking a box on tgstation.dme but I do not know how to approach it.

File: code/modules/mapfluff/ruins/spaceruin_code/transit_booth.dm [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/////////// transit_booth area
+/area/ruin/space/has_grav/transit_booth
+	name = "transit_booth"
+	icon = 'icons/area/areas_ruins.dmi'
+	icon_state = "ruins"
+	requires_power = FALSE
+	ambientsounds = list('sound/ambience/ambigen13.ogg','sound/ambience/ambigen14.ogg','sound/ambience/ambinice.ogg')
```
Comment (Maurukas) [inline code/modules/mapfluff/ruins/spaceruin_code/transit_booth.dm:7]: You can put this in `code/game/area/areas/ruins/space.dm`, just tack it on the end. This directory would be used for custom objects in the ruin.

File: tgstation.dme [hunk 3]
```diff
@@ -3698,6 +3698,7 @@
 #include "code\modules\mapfluff\ruins\spaceruin_code\originalcontent.dm"
 #include "code\modules\mapfluff\ruins\spaceruin_code\spacehotel.dm"
 #include "code\modules\mapfluff\ruins\spaceruin_code\TheDerelict.dm"
+#include "code\modules\mapfluff\ruins\spaceruin_code\transit_booth.dm"
```
Comment (Maurukas) [inline tgstation.dme:3701]: After you've moved the area you don't need this.

**Review by Maurukas — CHANGES_REQUESTED** [review]

**Review by Maurukas — APPROVED** [review]

**Pull request merged**